lib/models/LocationStatus.ts

```diff
@@ -0,0 +1,150 @@
+export type serviceStatus = {
+    paused: boolean,
+    scheduledResume: string|null,
+};
+
+export type LocationServiceStatus = {
```

Comment:
this is not the *Service* status : more like the overall location status (just like the class...)

→ maybe just `locationStatus`, or declare the type inside the LocationStatus class ?
→ do we need to export these 2 types ?